benches/oneshot/ipa.rs

```diff
@@ -134,3 +139,15 @@ async fn main() -> Result<(), Error> {
     );
     Ok(())
 }
+
+fn main() -> Result<(), Error> {
+    let args = Args::parse();
+    let rt = Builder::new_multi_thread()
+        .worker_threads(args.threads)
+        .enable_all()
+        .build()
+        .unwrap();
+    let _guard = rt.enter();
+    let task = rt.spawn(run(args));
+    rt.block_on(task)?
```

Comment:
Note that calling `block_on` here will ensure that the join task runs on a single thread, but the [documentation for `spawn`](https://docs.rs/tokio/latest/tokio/runtime/struct.Runtime.html#method.spawn) claims that the work will be moved to different threads as necessary, so that won't be the issue.